server/service/osquery.go

```diff
@@ -1166,15 +1168,36 @@ func processCalendarPolicies(
 	}
 
 	go func() {
-		if err := fleet.FireCalendarWebhook(
-			team.Config.Integrations.GoogleCalendar.WebhookURL,
-			host.ID, host.HardwareSerial, host.DisplayName(), failingCalendarPolicies, "",
-		); err != nil {
+		retryStrategy := backoff.NewExponentialBackOff()
+		retryStrategy.MaxElapsedTime = 30 * time.Minute
```

Comment:
I am assuming that there is another place in the code that checks that we are not firing outside the limits of the scheduled downtime (?)

Reply:
Yes, the initial webhook will not fire outside the scheduled downtime.